src/main/java/tempest/GUI/components/ActionButtonPanel.java

```diff
@@ -34,13 +34,15 @@ public JPanel getButtonPanel(GUIManager manager, Page page) {
         enterButton.setFocusable(false);
         enterButton.addActionListener(page);
 
-        buttonPanel.add(getCancelButton());
+        buttonPanel.add(getCancelButton(manager));
         buttonPanel.add(enterButton);
 
         return buttonPanel;
     }
 
-    public JButton getCancelButton() {
+    public JButton getCancelButton(GUIManager manager) {
+        this.manager = manager;
```

Comment:
Not sure I'm super comfortable with this side effect.

Reply:
I can create a separate CancelButton class? - getButtonPanel() would still call getCancelButton() (just it's now in a separate class)

Reply:
Yeah, that seems like a cleaner solution.